Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -6907,19 +6933,19 @@ static NSTextAlignment nsTextAlignmentFromTextAlignment(TextAlignment textAlignm
 
 #if ENABLE(IMAGE_ANALYSIS)
 
-CocoaImageAnalyzer *WebViewImpl::ensureImageAnalyzer()
+RetainPtr<CocoaImageAnalyzer> WebViewImpl::ensureProtectedImageAnalyzer()
 {
     if (!m_imageAnalyzer) {
         m_imageAnalyzerQueue = WorkQueue::create("WebKit image analyzer queue"_s);
         m_imageAnalyzer = createImageAnalyzer();
```

Comment:
Should likely use `lazyInitialize()` here and mark the data member as `const`, assuming it never gets reset.